changes for 2.35 (#994)
percona/pmm-doc: docs/get-started/backup/mongo-prerequisites.md

3. For setups where PMM Server runs as a Docker container, enable backup features at container creation time by adding `-e ENABLE_BACKUP_MANAGEMENT=1` to your `docker run` command.
4. Check that [PMM Client](../../setting-up/client/index.md) is installed. For creating logical backups, the PMM client should run on at least one node of the replica set. Make sure this is the one that will be used for backup and restore jobs. For physical backups, make sure PMM client runs on all nodes instead.  
5. Check that [Percona Backup for MongoDB](https://docs.percona.com/percona-backup-mongodb/index.html) (PBM) is installed and `pbm-agent` is running on all MongoDB nodes in the replica set. PMM 2.32 and later require PBM 2.0.1 or newer.

> 6. Check that your MongoDB Services are managed as a cluster. Go to **PMM Inventory > Services** page and make sure that all the services in the table specify a cluster name in the **Other Details** column. Services that do not specify a cluster name should be removed and re-added using the following command:

Reviewer: ```suggestion
6. Check that your MongoDB Services are managed as clusters in PMM. Go to **PMM Inventory > Services** page and make sure that all the services in the table specify a cluster name in the **Other Details** column. Services that do not specify a cluster name should be removed and re-added using the following command:
```

       <pre>pmm-admin add mongodb
       username=pmm_mongodb --password=password 

>        query-source=profiler <mark>--cluster=mycluster</mark></code></pre>

Reviewer: It's hard to find command in the article text. Can we highlight it with frame or something?

Author: It renders differently on the doc website. check out the [preview ](https://two-34-0-pr-994.onrender.com/get-started/backup/mongo-prerequisites.html)

> 7. Check that MongoDB is a member of a replica set.

Reviewer: ```suggestion
7. Check that MongoDB nodes are members of replica set.
```
